packages/core/src/node/messaging/messaging-contribution.ts

```diff
@@ -173,7 +173,7 @@ export class MessagingContribution implements BackendApplicationContribution, Me
 
     protected createChannel(id: number, socket: ws): WebSocketChannel {
         return new WebSocketChannel(id, content => {
-            if (socket.readyState < ws.CLOSING) {
+            if (socket.readyState === ws.OPEN) {
```

Comment:
An error often happened in the tests from that piece of code, so I quick fixed it.

It seems like websockets [won't send anything if it is not open](https://github.com/websockets/ws/blob/master/lib/websocket.js#L321).